odoo/addons/base/views/res_partner_views.xml

```diff
@@ -85,6 +85,17 @@
                     <field name="type" invisible="1"/>
                     <field name="avatar_128" invisible="1"/>
                     <field name="user_id" invisible="1"/>
+                    <field name="title" invisible="1"/>
```

Comment:
Linting: would it be possible to order alphabetically ? When having lots of fields, it helps finding if the one you are looking for is present ^^

Reply:
yes, indeed, thanks for reminding me.